known-good/hikey/2018-02-26_20:07:22-pinned-manifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
-  <remote fetch=".." name="aosp" review="https://android-review.googlesource.com/"/>
+  <remote fetch="https://android.googlesource.com/" name="aosp" review="https://android-review.googlesource.com/"/>
```

Comment:
Again, I'd rather not modify the manifests we fetched. I'd prefer we just remove the bad manifests and hopefully update the build process manifest creation to avoid the .. reference.